bots/elasticsearch/util.py

```diff
@@ -143,6 +145,29 @@ def fetch_creativework(pks):
                                         FROM share_agentidentifier AS identifier
                                         WHERE identifier.agent_id = agent.id
                                         ) AS identifiers ON TRUE
+                            LEFT JOIN LATERAL (
+                                        SELECT json_agg(json_strip_nulls(json_build_object(
+                                                                            'id', affiliated_agent.id
+                                                                            , 'type', affiliated_agent.type
+                                                                            , 'name', affiliated_agent.name
+                                                                            , 'affiliation_type', affiliation.type
+                                                                        ))) AS affiliations
+                                        FROM share_agentrelation AS affiliation
+                                        JOIN share_agent AS affiliated_agent ON affiliation.related_id = affiliated_agent.id
+                                        WHERE affiliation.subject_id = agent.id
+                                        ) AS affiliations ON TRUE
+                            LEFT JOIN LATERAL (
+                                        SELECT json_agg(json_strip_nulls(json_build_object(
+                                                                            'id', award.id
+                                                                            , 'name', award.name
+                                                                            , 'description', award.description
+                                                                            , 'uri', award.uri
+                                                                            , 'amount', award.award_amount
+                                                                        ))) AS awards
+                                        FROM share_throughawards AS throughaward
+                                        JOIN share_award AS award ON throughaward.award_id = award.id
+                                        WHERE throughaward.funder_id = agent_relation.id
```

Comment:
It is indexed on the work, in `lists.funders.awards`. All these changes are in `fetch_creativework`, I didn't touch `fetch_agent`.